harbor-helm/values.yaml

```diff
@@ -691,3 +691,34 @@ redis:
     password: ""
   ## Additional deployment annotations
   podAnnotations: {}
+
+# Tests that can be executed by using helm test
+tests:
+  api:
+    # Enable/disable API tests
+    enabled: true
+    # Stops test execution if any test fails
+    abortOnFailure: false
+    # Select test cases to run by tag
+    whitelist: []
+    # Select test cases not to run by tag. These tests are not run
+    # even if included on the whitelist.
+    # Note: Some tests are automatically skipped according to the
+    # harbor enabled services/features.
+    blacklist: []
+    image:
+      repository: registry.suse.de/home/flaviosr/branches/devel/caps/registry/containers/registry/harbor-test
+      tag: 2.0.0
+    # resources:
+    #  requests:
+    #    memory: 128Mi
+    #    cpu: 100m
+    dind:
+      image:
+        repository: registry.suse.de/home/flaviosr/branches/devel/caps/registry/containers/registry/docker-dind
+        tag: 19.03.11
```

Comment:
We should probably do the same we did with other images and refer the harbor version instead.

Reply:
done.